sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/yandex.yaml

```diff
@@ -0,0 +1,50 @@
+provider:
+  metadata:
+    name: Yandex
+    category: Cloud provider
+
+  rules:
+    - id: yandex_oauth_token
+      rspecKey: S6760
+      metadata:
+        name: Yandex OAuth tokens should not be disclosed
+        message: Make sure this Yandex OAuth token gets revoked, changed, and removed from the code.
+      detection:
```

Comment:
Detection can be improved by adding some post checks for dummy words like "test" or "example" (the CONTRIBUTING.md draft has a list of checks that are useful.)

Reply:
I have added `patternNot` at the level of the provider and some tests for those cases.